### ISSUE TITLE
reduce_sum cannot be performed on axis=0

### ISSUE THREAD

**Issue by MS-NLP** [Microsoft/CNTK#1280]
I'am building a very simple model for linear regression, trying to use the mean-square-loss as follows:

`loss = C.ops.reduce_sum(C.ops.squared_error(prediction, label)) / 2`

After feeding the computational graph with input data, both the "prediction" and "label" are of shape (sample_id, 1, 1), where sample_id ranges over the samples in the current mini_batch. However, the shape of "loss" is (sample_size) NOT (1). I then add a second argument "0" for reduce_sum, but failed.

Can anybody tell me how to correctly use the mean-square-loss? Thanks.

**Comment by sayanpa** [comment]
Would really appreciate if you could repost your question on stackoverflow.com. Have you tried

`reduce_mean(square(model - q_target), axis=0)`
